test/Twig/Tests/IntegrationTest.php

```diff
@@ -307,3 +307,43 @@ public function __toString()
         throw new Exception('__toString shall not be called on \Countables');
     }
 }
+
+/**
+ * This class is used in tests for the length filter
+ */
+class TraversableStub implements \Iterator
```

Comment:
Instead of creating a class, could we use the `ArrayIterator` PHP built-in class instead?